zypp/sat/Pool.cc

```diff
@@ -119,6 +119,16 @@ namespace zypp
     Pool::SolvableIterator Pool::solvablesEnd() const
     { return SolvableIterator(); }
 
+    sat::Queue Pool::whatMatchesDep(const SolvAttr &attr, const Capability &cap) const
+    {
+      std::list<Solvable> matchedSolvables;
```

Comment:
Appears to be unused.

Reply:
Leftover of how the code previously worked, removed